adit/settings/base.py

```
@@ -350,9 +350,7 @@
     },
     "dbbackup": {
         "BACKEND": "django.core.files.storage.FileSystemStorage",
-        "OPTIONS": {
-            "location": env.str("DBBACKUP_STORAGE_LOCATION", default="/tmp/backups-adit")
-        },
+        "OPTIONS": {"location": env.str("DBBACKUP_STORAGE_LOCATION", default="/tmp/backups-adit")},
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/backups-adit"

(S108)